### PR TITLE
Blog app - views

### PR DESCRIPTION
**Pull request by shyusu4**
## PR Description

- Used preprocessed HTML file with embedded Ruby code.
- Implemented views.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,38 @@
...
+            </div>
+        <div>
+        </a>
+        <div class="user-comment__item">
+                <% post.comments.each do |comment| %>
+                <div class="user-comment__content">
+                            <b><%= User.find(comment.author_id).name %></b>
+                            <span class="comment-date"><%= comment.created_at %></span>
+                </div>
+                            <p class="comment"><%= comment.text %></p>
```
Comment (elisha2kyakpo1) [inline app/views/posts/index.html.erb:30]: - [ ] Let's use our model methods to render 5 recent comments for a post. 

Hint: 
 ```
<% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @shyusu4, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look excellent and professional.
✔️ No linter errors.
✔️ You have descriptive commit messages.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (shyusu4) [reply app/views/posts/index.html.erb:30]: @elisha2kyakpo1 Thank you for the review! I'll make sure to refactor my code according to your requested changes.

**Review by fobadara — APPROVED** [review]
Hi @shyusu4, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**